.eslintrc

```diff
@@ -1,58 +1,55 @@
 {
-    "rules": {
-        "indent": [
-            2,
-            2
-        ],
-        "quotes": [
-            2,
-            "single"
-        ],
-        "linebreak-style": [
-            2,
-            "unix"
-        ],
-        "semi": [
-            2,
-            "always"
-        ],
-        "strict": 0,
-        "react/display-name": [1, { "acceptTranspilerName": true }],
-        "react/jsx-boolean-value": [1, "never"],
-        "react/jsx-curly-spacing": [1, "never"],
-        "react/jsx-no-duplicate-props": 1,
-        "react/jsx-no-undef": 1,
-        "react/jsx-quotes": 1,
-        "react/jsx-sort-prop-types": 1,
-        "react/jsx-sort-props": 0,
-        "react/jsx-uses-react": 1,
-        "react/jsx-uses-vars": 1,
-        "react/no-danger": 1,
-        "react/no-did-mount-set-state": 1,
-        "react/no-did-update-set-state": 1,
-        "react/no-multi-comp": 0, // reconsider this later
-        "react/no-unknown-property": 1,
-        "react/prop-types": 1,
-        "react/react-in-jsx-scope": 1,
-        "react/require-extension": 1,
-        "react/self-closing-comp": 1,
-        "react/sort-comp": 1,
-        "react/wrap-multilines": 1
-    },
-    "env": {
-        "es6": true,
-        "browser": true,
-        "jasmine": true,
-        'node': true,
-        "jest": true
-    },
-    "extends": "eslint:recommended",
-    "ecmaFeatures": {
-        "jsx": true,
-        "experimentalObjectRestSpread": true
-    },
-    "plugins": [
-        "react"
-    ],
-    "parser": "babel-eslint"
+  "rules": {
+    "default-case": 0,
+    "indent": [ 2, 2 ],
```

Comment:
Just a note: If we enforce 2 space indentation, it would make sense to add an .editorconfig (http://editorconfig.org/) file as well. This could make life a lot easier for contributers with 4-space settings.